config/ighn.international.uiowa.edu/user.role.publisher.yml

```diff
@@ -64,12 +64,15 @@ permissions:
   - 'create terms in research_areas'
   - 'create terms in tags'
   - 'create twitter media'
-  - 'create url aliases'
```

Comment:
Was removing this intentional?

Reply:
ahhh forgot to push up fix to that!  Haha 100% was not.